src/ia-sharing-options.js

```diff
@@ -104,11 +106,14 @@ export class IASharingOptions extends LitElement {
     this.embedOptionsVisible = !this.embedOptionsVisible;
   }
 
+  get header() {
+    const header = html`<header><h3>Share this ${this.type}</h3></header>`;
```

Comment:
I thought the header was being removed from individual submenu components and instead rendered by the menu slider component? Rendering this header within a shadowDOM like this means the close button and any other buttons meant to sit alongside the header have the potential to be overlapped by this header element.

Reply:
they are :)
i am preserving the header for stand-alone usage